tools/setup_helpers/extension.py

```diff
@@ -37,6 +37,7 @@ def _get_build(var, default=False):
 _BUILD_KALDI = False if platform.system() == "Windows" else _get_build("BUILD_KALDI", True)
 _BUILD_RIR = _get_build("BUILD_RIR", True)
 _BUILD_RNNT = _get_build("BUILD_RNNT", True)
+_BUILD_CUDA_CTC_DECODER = _get_build("BUILD_CUDA_CTC_DECODER", True)
```

Comment:
I think this is better dependent on the value of `_USE_CUDA`. Can you move this line after `_USE_CUDA`?

```suggestion
_BUILD_CUDA_CTC_DECODER = _get_build("BUILD_CUDA_CTC_DECODER", _USE_CUDA)
```